### ISSUE TITLE
回答不完整

### ISSUE THREAD

**Issue by SmallNancy** [ohmplatform/FreedomGPT#63]
![15953566f8a2896d7b22b993ca7b18c](https://user-images.githubusercontent.com/17695995/234481612-fef869a6-6815-4e22-b68c-27de80be50b0.png)

**Comment by superluig164** [comment]
I get this as well, on all models   (windows 11).

**Comment by undefined-bob** [comment]
Same here. This is the most annoying thing which makes it kinda unusable (beside the fact that's it's very slow)

**Comment by superluig164** [comment]
I mean yeah I could live with it being slow if I could just leave it and come back to a completed prompt.  But it just stops after like 3 sentences.

**Comment by undefined-bob** [comment]
@superluig164 same for me. But I've seen others getting much longer answers. So I think our issue is maybe related to slower CPUs. I have only an i5-7300U with 16GB. But I also think it's something they can fix in the code

**Comment by junkse7en** [comment]
Yeah, I only get a max of four lines. Normally it cuts off mid sentence. In this state, for me, it is unusable, at least in any meaningful way.

**Comment by Abbodarr** [comment]
Same problem here. I get a couple lines before the output cuts out.  Win11,  i7-9750H , 16 GB RAM.

**Comment by VirusGames** [comment]
The same here, i5-7300HQ and 16 GB of RAM. I can get only 4/5 lines before it cuts off.

**Comment by peter-alexander** [comment]
This affects me too. Windows 11 22H2, i7-4610M, 16GB
